src/eradiate/test_tools/regression.py

```diff
@@ -62,17 +75,50 @@ def regression_test_plots(
 
     axes[0][1].set_axis_off()
     axes[0][1].legend(handles=handles, labels=labels, loc="upper center")
-    axes[0][1].text(
-        0.5, 0.5, f"{metric[0]}: {metric[1]:.4}", horizontalalignment="center"
-    )
+
+    if metric[1] is None:
+        axes[0][1].text(
+            0.5,
+            0.5,
+            f'Metric "{metric[0]}" is not available',
+            horizontalalignment="center",
+        )
+    else:
+        axes[0][1].text(
+            0.5, 0.5, f"{metric[0]}: {metric[1]:.4}", horizontalalignment="center"
+        )
 
     plt.tight_layout()
-    plt.savefig(filename)
-    plt.close()
+
+
+def render_svg_chart():
+    str_i = StringIO()
+    plt.savefig(str_i, format="svg", transparent=True)
```

Comment:
Just to avoid having text cut off:
```suggestion
    plt.savefig(str_i, format="svg", transparent=True, bbox_inches="tight")
```